desktop/cmp/rest/RestGridModel.js

```diff
@@ -124,6 +124,7 @@ export class RestGridModel {
         enhanceToolbar,
         editors = [],
         store,
+        enableExport = true,
```

Comment:
I don't think we need this here - we can just have `enableExport: true` (right before our default `exportOptions`) below when creating the GridModel, and if the app explicitly set it to false when creating their RestGridModel then it would override our default via the `...rest` destructuring